### DISCUSSION TITLE
Good Reads (share references / publications)

### DISCUSSION THREAD

**Discussion by arthurrge** [aiaa-deic/computation-ai#6] [Show and tell]
1. [Vision for Computational Engineering](https://drive.google.com/file/d/1Jqg5nsf5iCf4R18zKAFJ0y6-o9RY7Wol/view?usp=share_link) (to ASSESS Mar-2023)
2. [The Modeling Imperative](https://drive.google.com/file/d/1DfcxN2IV7zbn4jHd7m7qXwebDS3qPFQC/view?usp=share_link) (to FNL, Oct-2021)
3. [Modeling Maturity Rubric](https://drive.google.com/file/d/19URWTkIOJ01j62dqnNjIQrlQv_pnEX0J/view?usp=share_link)

**Comment by arthurrge** [comment]
[(May 2023) "Digital Literacy and the Engineer of the Future" @Dayton Digital Transformation Summit](https://drive.google.com/file/d/1ywzw534y85FgjQY7S9NUrElQ2xLjnRFz/view)

**Comment by smarnold59** [comment]
Arthur thanks for providing these past resources -- very nice

**Comment by arthurrge** [comment]
I have recently pulled together an attempt at an exec-friendly briefing on Quantum Computing, which we've been tracking since the early 2000s in then-collaborations with Lockheed. That topic seems to light a brush fire a few times a year that requires some attention to recalibrate the execs reacting to a news article. This is the sort of thing I believe would be in scope for tracking as part of this subcommittee - and it has been the subject of a number of SciTech papers, etc. I am unsure of where else it may already be "included" within AIAA.

**Comment by arthurrge** [comment]
[QC-Briefing-clean.pdf](https://github.com/aiaa-deic/computation-ai/files/14115562/QC-Briefing-clean.pdf)

**Comment by smarnold59** [comment]
Thanks Rick

__________________________________________
Steven M. Arnold, PhD, FASM
Technical Lead: Multiscale Multiphysics Modeling
T3 Project: Material and Structures Discipline Lead
Materials and Structures Division
NASA Glenn Research Center at Lewis Field
21000 Brookpark Rd.  MS 49-7
Cleveland, OH  44135


Ph: 216-433-3334    Fax: 216-433-8300
Email: ***@***.***
Chairman MDMC: www.mdmc.net


From: Rick Arthur ***@***.***>
Reply-To: aiaa-deic/computation-ai ***@***.***>
Date: Friday, February 2, 2024 at 8:25 AM
To: aiaa-deic/computation-ai ***@***.***>
Cc: Steven Arnold ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] [BULK] Re: [aiaa-deic/computation-ai] Prior Talks (Discussion #6)

CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.



See: National Academies' Charting a Path in a Shifting Technical and Geopolitical Landscape<https://nap.nationalacademies.org/read/26916> / DOI 10.17226/26916<https://nap.nationalacademies.org/catalog/26916/charting-a-path-in-a-shifting-technical-and-geopolitical-landscape>

**Comment by arthurrge** [comment]
There are numerous great topics from technical exchange meetings MITRE has held on behalf of the FAA relating to **AI/ML Safety Assurance**. I'm sure some of you have even attended - but listing here the archive index of the talks, where you can then click to view the presentation material: [https://na.eventscloud.com/ereg/inactive.php?eventid=768017](https://na.eventscloud.com/ereg/inactive.php?eventid=768017)
